frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -72,12 +73,15 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
 
   const subscription = resources?.subscription as FirehoseResult;
   const subscriptionLoaded = subscription?.loaded;
-  const ocsVersion = getOCSVersion(subscription);
-  const ocsPath = `${resourcePathFromModel(
+  const serviceVersion = !isODF ? getOCSVersion(subscription) : getODFVersion(subscription);
+  const servicePath = `${resourcePathFromModel(
     ClusterServiceVersionModel,
-    ocsVersion,
+    serviceVersion,
     CEPH_STORAGE_NAMESPACE,
   )}`;
+  const providerName = isODF
```

Comment:
```suggestion
  const serviceName = isODF
```
To be consistent with other files. 
Plus the attribute below seems to be service name too